Sources/TuistKit/Services/EditService.swift

```diff
@@ -52,6 +52,10 @@ final class EditService {
     ) throws {
         let path = self.path(path)
 
+        let pathsToExclude = [
+            "**/\(Constants.tuistDirectoryName)/\(Constants.DependenciesDirectory.name)/**",
```

Comment:
Do we need the initial **/?

Reply:
I think so because the `tuist edit` command can be executed from directories other than the root. So we can't assume we'lll be in the root directory.

Reply:
👍